db/index.js

```diff
@@ -55,18 +55,18 @@ seedUsers()
   let queryString = 'select count(*) as count from users';
   connection.query(queryString, (err, result) => {
     if (err){console.log(err)}
-    if (result[0].count === 100) {
+    if (result[0].count >= 100) {
```

Comment:
I think closing the connection based on the count may not work because these functions are asynchronous. I'm still trying to figure out how to close my connection after seeding and will let you know if I come up with a solution, though maybe Anthony or Kaz have some insight on this?